src/components/Comments/CommentItem.tsx

```diff
@@ -19,9 +19,27 @@ export const CommentItem: FC<{ comment: any }> = observer(({ comment }) => {
   const currentUser = window.APP_SETTINGS?.user;
   const initialComment = comment.text;
   const [currentComment, setCurrentComment] = useState(initialComment);
+  const [isEdited, setIsEdited] = useState<Date | boolean | undefined>();
 
   if (comment.isDeleted) return null;
+  
+  useEffect(() => {
+    if (comment.isEditMode) setIsEdited(false);
+    else if (isEdited === false) setIsEdited((new Date()));
+  }, [comment.isEditMode]);
 
+  const TimeTracker = () => {
+    const time = isEdited ? isEdited : comment.createdAt;
+  
+    return comment.isPersisted && comment.createdAt && (
+      <Elem name="date">
+        <Tooltip placement="topRight" title={new Date(time).toLocaleString()}>
+          {`${isEdited ? "edited" : "created"} ${humanDateDiff(time)}`}
```

Comment:
just a nitpick, but it seems redundant to put the word `created` for every comment. The specific clarification is the edited state, so I would think this would be better to just add `edited` when its the differing state.